abci/example/kvstore/kvstore.go

```diff
@@ -162,6 +169,9 @@ func isValidTx(tx []byte) bool {
 // quite a trivial example of transaction modification.
 // NOTE: we assume that CometBFT will never provide more transactions than can fit in a block.
 func (app *Application) PrepareProposal(ctx context.Context, req *types.RequestPrepareProposal) (*types.ResponsePrepareProposal, error) {
+	if app.generateBlobs {
+		return &types.ResponsePrepareProposal{Txs: app.formatTxs(ctx, req.Txs), Blob: []byte("hiBlob")}, nil
+	}
```

Comment:
Why use `hiBlob` here instead of what is done in the original PR where constant is used?

Reply:
The relevant commit was not cherry-picked yet. Now it is, this should be resolved.